pkg/piecestore/psserver/agreementsender/agreementsender.go

```diff
@@ -93,13 +94,17 @@ func (as *AgreementSender) sendAgreementsToSatellite(ctx context.Context, satID
 		// Send agreement to satellite
 		r, err := client.BandwidthAgreements(ctx, msg)
 		if err != nil || r.GetStatus() != pb.AgreementsSummary_OK {
-			as.log.Error("Agreementsender failed to send agreement to satellite", zap.Error(err))
-			return
+			if bwagreement.BwAgreementError.Has(err) {
```

Comment:
Should we include package that is specific for satellite? @egonelbre

Reply:
definitely not